docs/latest/design/ratelimit.md

```diff
@@ -0,0 +1,66 @@
+# Ratelimiting Design
```

Comment:
Envoy docs use `Global Rate Limit` and `Local Rate Limit`. For consistency, I suggest we do the same by using `Rate Limiting Design` in the title and `rate limit` or ` rate limiting` throughout.